### PR TITLE
Got end game broadcasting functionality working

### PR DESCRIPTION
**Pull request by Quickzand**

### CODE REVIEW

File: POOSDKer/DemonstrationView.swift
Comment (aintzanepm) [inline POOSDKer/DemonstrationView.swift:1]: Shouldn't we get rid of Demonstration View?

File: POOSDKer/Views/LobbyView.swift [hunk 1]
```diff
@@ -38,16 +38,22 @@ struct LobbyView: View {
...
         }
         .onDisappear {
-            if(appState.isHost) {
-                appState.networkingController?.stopHosting();
-            }
-            else {
-                appState.networkingController?.disconnectFromHost()
+            if(!appState.isInGame) {
+                if(appState.isHost) {
+
```
Comment (maximusmith529) [inline POOSDKer/Views/LobbyView.swift:43]: Why are these not following standard?

File: POOSDKer/Views/LobbyView.swift [hunk 2]
```diff
@@ -14,7 +14,7 @@ struct LobbyView: View {
     
     var body: some View {
         VStack {
- 
+
```
Comment (maximusmith529) [inline POOSDKer/Views/LobbyView.swift:17]: and this

**Review by maximusmith529 — CHANGES_REQUESTED** [review]

File: POOSDKer/Views/LobbyView.swift (hunk 1 above)
Comment (aintzanepm) [reply POOSDKer/Views/LobbyView.swift:43]: Do we have a set standard? Would love to have it on record for future PRs

File: POOSDKer/DemonstrationView.swift
Comment (Quickzand) [reply POOSDKer/DemonstrationView.swift:1]: Very good point!

**Review by maximusmith529 — COMMENTED** [review]
lgtm

**Review by aintzanepm — APPROVED** [review]
LGTM!

**Review by amymariag — APPROVED** [review]
looks good, let's try to keep the indentation standard.

**Pull request merged**